.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,39 @@
 <!-- 
 Hey, thanks for raising a PR! We're excited to see what you've done!
-To help us review the changes, please complete each section in this template by replacing '...' with details to help the reviewers of this pull request. 
+To help us review the changes, please complete the template below to help the reviewers of this pull request. 
 -->
 
 # Brief overview of changes
 
-...
+Replace this with a brief description of what changes have been made in this PR.
 
 ## Why are these changes being made?
 
-...
+Replace this with a description of why these changes are necessary.
 
 ## Detailed description of changes
 
-...
+Replace this with a detailed description of the changes made in this PR. Use bullet points if necessary to break down the changes.
 
-## Additional information for reviewers
+## Essential Checklist
 
-...
+- [ ] I have read the [contributing guidelines](https://dfe-analytical-services.github.io/dfeR/CONTRIBUTING.html)
+- [ ] The code follows the package style and naming conventions
+- [ ] All new and existing tests pass (`devtools::test()`)
+- [ ] I have checked that my changes do not break existing functionality
+
```

Comment:
Worth an extra line for 'I have updated the documentation using 'devtools::document()`'?